treeherder/log_parser/parsers.py

```diff
@@ -505,6 +505,8 @@ def parse_line(self, line, lineno):
         if match:
             try:
                 data = json.loads(match.group(1))
+                if not bool(data):
+                    raise EmptyPerformanceData("The perf data is empty.")
                 validate_perf_data(data)
```

Comment:
@armenzg we will still lose context on other forms of validation errors.

Reply:
How? This is *only* raised if PERFERHERDER_DATA is an empty object. In all other cases it will still hit `validate_perf_data(data)`.